lib/space.rb

```diff
@@ -18,7 +20,7 @@ def price_per_night_formatted
   def self.all
     spaces = []
     DatabaseConnection.query("SELECT * FROM spaces").each do |space|
-      spaces << Space.new(space['id'], space['name'], space['description'], Money.new(space['price']), Date.parse(space['available_from']), Date.parse(space['available_to']))
+    spaces << Space.new(space['id'], space['name'], space['description'], Money.new(space['price']), Date.parse(space['available_from']), Date.parse(space['available_to']), space['user_id'])
```

Comment:
the only tiny bit of feedback would be to put these on to separate lines for legibility